docs/agented-vm-guide.md

```diff
@@ -212,21 +212,41 @@ documentation.
 
 ### Installation On Linux VMs
 
-Download the [wrapper install script](../hack/install-wrapper.sh) and run
-installation.
+Download the [wrapper install script](../hack/install-vm-agent-wrapper.sh) from a
+[list of nephe releases](https://github.com/antrea-io/nephe/releases) and run
+the installation. For any given release <TAG> (e.g. v0.2.0), you can download
+the script as follows:
 
 ```bash
-./install-wrapper.sh --ns vm-ns --antrea-version v1.9.0 --kubeconfig ./antrea-agent.kubeconfig \
-  --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig --bin ./antrea-agent
+curl https://github.com/antrea-io/nephe/releases/download/<TAG>/install-vm-agent-wrapper.sh --output install-vm-agent-wrapper.sh
+```
+
+To download the latest version of wrapper install script, use the checked-in
+[wrapper install script](../hack/install-vm-agent-wrapper.sh)
+
+```bash
+curl https://raw.githubusercontent.com/antrea-io/nephe/main/hack/install-vm-agent-wrapper.sh --output install-vm-agent-wrapper.sh
+./install-vm-agent-wrapper.sh --ns vm-ns --antrea-version v1.9.0 --kubeconfig ./antrea-agent.kubeconfig \
```

Comment:
can you separate this and mention run the install script as, it seems only for main, we are running the install.

Reply:
Done